client/trino-jdbc/src/test/java/io/trino/jdbc/TestJdbcConnection.java

```diff
@@ -465,6 +465,29 @@ public void testConcurrentCancellationOnConnectionClose()
         testConcurrentCancellationOnConnectionClose(false);
     }
 
+    @Test
```

Comment:
Here we test if we can create a `PreparedStatement` with the options:
- `NO_GENERATED_KEYS`
- `RETURN_GENERATED_KEYS`
The content of the request allowing the creation of this `PreparedStatement` is of a minimal format.